awx/main/tasks/jobs.py

```diff
@@ -1451,8 +1451,8 @@ def build_execution_environment_params(self, instance, private_data_dir):
         params.setdefault('container_volume_mounts', [])
         params['container_volume_mounts'].extend(
             [
-                f"{project_path}:{project_path}:Z",
-                f"{cache_path}:{cache_path}:Z",
+                f"{project_path}:{project_path}:z",
```

Comment:
Can we make this a setting? I would love to keep `:Z` in those platforms that work with `:Z`

Reply:
@Zokormazo I think this is ok. It's not about the platform... this becomes a problem when we try to containerized the control plane services. `:Z` is the same as `:z` except it only allows 1 container to be accessing a directory.

Reply:
`Z` may cause unintended race condition?

if 2 container try to mount the same dir/file only the last container will retain access to the dir/file and rest of the container will get permission deny when try to read from and write to the file

Reply:
@Zokormazo are u ok with the change?

Reply:
Followed up with Julen in Slack to resolve this.